### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by manuelsh** [XifengGuo/CapsNet-Keras#55]
Hi,

Great implementation, congratulations. Do you have any idea why you don't arrive to the performance of the paper?

Thanks

Manuel

**Comment by XifengGuo** [comment]
@manuelsh See the `differences with paper` in README.md

**Comment by manuelsh** [comment]
Great, thanks! FYI, in my implementation I was able to arrive to the results of the paper, main thing was the decaying factor of the learning rate, which, if tuned finely, makes you get the result. See https://github.com/manuelsh/capsule-networks-pytorch

**Comment by XifengGuo** [comment]
Excellent!




Xifeng Guo
On 08/27/2018 22:56, Manuel wrote:

Great, thanks! FYI, in my implementation I was able to arrive to the results of the paper, main thing was the decaying factor of the learning rate, which, if tuned finely, makes you get the result. See https://github.com/manuelsh/capsule-networks-pytorch
